src/elf.rs

```diff
@@ -2092,10 +2100,10 @@ mod test {
         let config = Config::default();
         let elf_bytes = [0u8; 512];
 
-        let s1 = new_section(0, 10);
-        let s2 = new_section(20, 10);
-        let s3 = new_section(40, 10);
-        let s4 = new_section(50, 10);
+        let s1 = new_section(ebpf::MM_PROGRAM_START, 10);
```

Comment:
Instead of changing the existing test, wouldn't it be better to add a new test
with this "high" vaddrs? We have code that does one thing or another depending
on whether vaddr < MM_PROGRAM_START or >. I think we should test both?